### PR TITLE
Addind information #3198

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added important note with really important information given in #3198

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ This topic provides information that will help you diagnose and resolve problems
 
 - [The maximum number of concurrent shells for this tenant in Skype for Business Online has been exceeded](#the-maximum-number-of-concurrent-shells-for-this-tenant-in-skype-for-business-online-has-been-exceeded)
     
+
+> [!IMPORTANT]
+> Powershell sessions by default stand for sixty minutes, for reconnecting, you have to close and launch a new Powershell session to reconnect. Recently has been launched a new version of [Skype for Business Online, Windows PowerShell Module (2046.123 - Published 10/2/2019)](https://www.microsoft.com/en-us/download/details.aspx?id=39366) which brings a new Cmdlet called **Enable-CsOnlineSessionForReconnection** which mitigate the 60 minutes timeout issue.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md]:
```suggestion
> By default, PowerShell sessions time out after sixty minutes. To reconnect, you have to close the session and launch a new PowerShell session. A new version of [Skype for Business Online, Windows PowerShell Module (2046.123 - Published 10/2/2019)](https://www.microsoft.com/download/details.aspx?id=39366), has been launched recently which includes a new cmdlet called **Enable-CsOnlineSessionForReconnection** that mitigates the 60 minutes time-out issue.
```

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md [hunk 2]
```diff
@@ -44,6 +44,13 @@ This topic provides information that will help you diagnose and resolve problems
...
+> The PS session reconnects and authenticates, allowing it to be re-used without having to launch a new instance to reconnect.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md]:
```suggestion
> The PowerShell session reconnects and authenticates, allowing it to be re-used without having to launch a new instance to reconnect.
```

**Comment by nenonix** [comment]
@j0rt3g4 I'll do another sweep when the changes are applied.

**Comment by j0rt3g4** [comment]
sure @nenonix  they are already there. Thank you!

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**